tests/test_ToDoApp.py

```diff
@@ -0,0 +1,107 @@
+from todo_app.viewModel import taskCards
+from todo_app.ToDoItem  import  ToDoItem
+import pytest 
+
+from datetime import datetime
+
+
+statusList = ['To Do','Doing','Done','Test']
+
+def test_items_category_split():
+	items = [
+		ToDoItem(statusList,{'id':1, 'idList':0,'name':'task 1','due':"2021-08-27T00:00:00.000Z",'dateLastActivity':"2021-08-20T00:00:00.000Z"}),
+		ToDoItem(statusList,{'id':2, 'idList':1,'name':'task 2','due':"2021-08-27T00:00:00.000Z",'dateLastActivity':"2021-08-20T00:00:00.000Z"}),
+		ToDoItem(statusList,{'id':3, 'idList':2,'name':'task 3','due':"2021-08-27T00:00:00.000Z",'dateLastActivity':"2021-08-20T00:00:00.000Z"}),
+		ToDoItem(statusList,{'id':4, 'idList':3,'name':'task 4','due':"2021-08-27T00:00:00.000Z",'dateLastActivity':"2021-08-20T00:00:00.000Z"}),
+		ToDoItem(statusList,{'id':5, 'idList':2,'name':'task 5','due':"2021-08-27T00:00:00.000Z",'dateLastActivity':"2021-08-20T00:00:00.000Z"}),
+		ToDoItem(statusList,{'id':6, 'idList':1,'name':'task 6','due':"2021-08-27T00:00:00.000Z",'dateLastActivity':"2021-08-20T00:00:00.000Z"})
+	]	
+
+	view = taskCards(items)
+	
+	todo_items_tasks=[]
+	for card in view.todo_items:
+    		todo_items_tasks.append(card.title)
+	assert todo_items_tasks == ['task 1']
```

Comment:
Minor, but we could capture this slightly more succinctly with e.g.:
```suggestion
	assert len(view.todo_items) == 1
	assert view.todo_items[0].title == 'task 1'
```